lib/themes/dosomething/paraneue_dosomething/template.php

```diff
@@ -92,6 +94,7 @@ function paraneue_dosomething_js_alter(&$js) {
 
   // Force removal of files that we can't seem to kill with exclude[js] array
   unset($js['profiles/dosomething/modules/contrib/devel/devel_krumo_path.js']);
+  unset($js['profiles/dosomething/modules/contrib/google_analytics/googleanalytics.js']);
```

Comment:
Yep! The actual tracking snippet is added there as an inline script (which then makes its own async request for Google's analytics code from their CDN).